artist.py

```diff
@@ -2,42 +2,49 @@
 import os
 from common_constants import Vector, COLORS
 
+
 # in charge of drawing Game objects
 class Artist:
     def __init__(self, game):
         self.game = game
 
+        # locate the window in the upper left
+        # os.environ['SDL_VIDEO_WINDOW_POS'] = "%d,%d" % (0, 0)
         # pygame setup
-        os.environ['SDL_VIDEO_WINDOW_POS'] = "%d,%d" % (0,0) # locate the window in the upper left
         pygame.init()
         pygame.display.set_caption(f'{type(game).__name__} Reduction')
 
         # compute constants
 
         # this is the approximate height of the menu bar at the top of iOS
-        BUFFER = 28
+        BUFFER = 280
```

Comment:
Was the graphical stuff here important to you? Putting the window in the corner cut off the minimize/maximize/close which made it a pain to exit.

Reply:
this is to make sure the window is fully within the screen at large window sizes, but it sounds like we need a window position buffer to make sure the control bar at the top of the window is accessible on systems like yours

Reply:
i'm going to go ahead with the pull request, and maybe you can add the window position buffer later when you get a chance, it's obviously not mission-critical